kitsune/search/v2/management/commands/es7_reindex.py

```diff
@@ -53,4 +72,7 @@ def handle(self, *args, **kwargs):
                 start = x * bulk_count
                 end = start + bulk_count
                 index_objects_bulk.delay(dt.__name__, id_list[start:end])
+                if kwargs["print_sql_count"]:
+                    print("{} SQL queries executed".format(len(connection.queries)))
+                    reset_queries()
```

Comment:
`reset_queries()`? it clears the contents of `connections.queries`, so we're only counting the number of queries executed in that batch, rather than the running total.